src/controllers/productController.ts

```diff
@@ -99,7 +96,7 @@ async function destroyProduct(req: Request, res: Response, next: NextFunction) {
   }
 }
 
-export const indexProductFuncs = [indexProduct, getMultipleIdFormatting];
+export const indexProductFuncs = [indexProduct, showProduct];
 export const showProductFuncs = [retrieveProduct, getIdFormatting, showProduct];
 export const createProductFuncs = [postIdFormatting, createProduct];
```

Comment:
Could probably just remove the `getIdFormatting` and `postIdFormatting` for consistency in the API responses. Can first comment out their respective functions above too.

Reply:
Removed